docs/src/tutorial/profiling_tutorial.rst

```diff
@@ -75,7 +75,7 @@ Enabling coverage analysis
 --------------------------
 
 Since Cython 0.23, line tracing (see above) also enables support for coverage
-reporting with the `coverage.py <http://coverage.readthedocs.io/>`_ tool.
+reporting with the `coverage.py <https://coverage.readthedocs.io/en/v4.5.x/>`_ tool.
```

Comment:
We shouldn't include the version here, just let the server send its redirect to whatever it wants.